packages/EasyCodingStandard/config/config.yaml

```diff
@@ -9,7 +9,7 @@ imports:
 parameters:
     # defaults for "PhpCsFixer\WhitespacesFixerConfig"
     indentation: 'spaces'
-    line_ending: !php/const PHP_EOL
+    line_ending: \n
```

Comment:
If anyone is using Windows, this would break their lines now

Reply:
Well, it goes the other way too. If someone on a team is working on windows and the project uses LF, that will cause the breaking.

Is there another way to fix it?

Reply:
Here should be the default value of `PHP_EOL`, so it respects native OS values.

If someone is using Linux endlines on Windows or Windows endlins on Linux, they have to specify it in their config.

Reply:
Ok. What does it means for the tests of Symplify on windows. Do I have to create a specific config.yaml and specific composer command who runs the tests with an alternative config file?

Reply:
> Do I have to create a specific config.yaml and specific composer command who runs the tests with an alternative config file?

Yes, that would be the way to go

Reply:
Might be relevant to ask there: https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/3955#issuecomment-409673885

Reply:
Not sure I'll have time to do that anytime soon. Maybe this could be merged in the current state for now?